### DISCUSSION TITLE
Testing the AP2 build !

### DISCUSSION THREAD

**Discussion by acresp** [mikebrady/shairport-sync#1228] [General]
Hello,
First, I have to say a big thanks to all the guys who helped to make this possible.
I follow the AirPlay 2 support thread since a long time and it's very cool (and surprising) to have a dev release which is very strong.

To begin, I took 2/5 of my Pi's to the 4.0-dev version, and seriously, this is surprising (saying again :p)

![IMG_8282](https://user-images.githubusercontent.com/57019575/125193304-10ab8480-e24c-11eb-9ce5-3b5bf4513e1a.PNG)

Because I use the Pi's also for webradios (with Volumio and some ESP32 with push buttons in my walls), I didn't begin with a "fresh" RPi OS. I tested with the Volumio "buster" distrib but had lot of issues about dependencies with some packages (libavcodec, etc.). So I tried an other one, MoOde Audio. I was able to compile & install nqptp & shairport-sync on the last revision without any particular issue. And the good part: all the "shairport" tweaks like the name, enable/disable through the GUI is working well.

I will test and follow the enhancements, and If I find some strange things I will post a feedback :-)

Again, thanks a lot!

And a picture of the setup in the cabinet:

![IMG_8283](https://user-images.githubusercontent.com/57019575/125193470-d1c9fe80-e24c-11eb-9c7e-df6c54a0685b.JPG)

And here the beast opened (5 Pi's + HiFiBerry AMP2 + power supply + switch + fuses board):

![IMG_6662](https://user-images.githubusercontent.com/57019575/125193513-00e07000-e24d-11eb-9e2c-46ef268eb94c.jpg)

**Comment by charlesomer** [comment]
I'm very jealous of your setup 😄 Nice!

**Comment by mikebrady** [comment]
Very nice, @acresp. Thanks for testing.
